.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,87 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Bug Report
+description: File a bug report for cuDF
+title: "[BUG]: "
+labels: ["bug"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuDF and fill out this bug report!
+        - By submitting this issue, you agree to follow our [Code of Conduct](https://docs.rapids.ai/resources/conduct/)
+        - You also confirm that you have searched the [open bugs](https://github.com/rapidsai/cuDF/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this request
```

This will redirect. Use `cudf` as the repo name, not `cuDF`.
```suggestion
        - You also confirm that you have searched the [open bugs](https://github.com/rapidsai/cudf/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this request
```